code/modules/power/turbine/turbine_parts.dm

```diff
@@ -0,0 +1,84 @@
+/obj/item/turbine_parts
+	name = "turbine parts"
+	desc = "you really should call an admin"
+	icon = 'icons/obj/turbine/turbine.dmi'
+	icon_state = "inlet_compressor"
+
+	var/part_efficiency = 0
+	var/part_efficiency_increase_amount = 0
+
+	var/current_tier = 1
+	var/max_tier = 4
+
+	var/second_tier_material = /obj/item/stack/sheet/plasteel
+	var/second_tier_material_amount = 10
+
+	var/third_tier_material = /obj/item/stack/sheet/mineral/titanium
+	var/third_tier_material_amount = 10
+
+	var/fourth_tier_material = /obj/item/stack/sheet/mineral/metal_hydrogen
+	var/fourth_tier_material_amount = 5
+
+	var/max_rpm = 35000
+	var/max_rpm_tier_multiplier = 2.5 //max should be 500000 rpm
```

Comment:
Math does not add up if i'm doing it right. maybe make something easier like 50.000/200.000/350.000/500.000

Reply:
it's fine, it doesn't need to be 500000 precisely, is more like a guideline